hack/debug.sh

```diff
@@ -61,6 +70,16 @@ log_system () {
 }
 
 do_master () {
+    services=($aos_master_service $aos_master_controllers_service $aos_master_api_service)
+    start_args=("master" "master controllers" "master api")
+    config_files=()
+    for i in "${!services[@]}"; do
+        config_files[$i]=$(get_config_path_from_service "${start_args[$i]}" "${services[$i]}")
+        if [ -z "${config_files[$i]}" ]; then
+            config_files[$i]="zzzz"
```

Comment:
You could use associative sub-arrays in bash:
service_config_files = ( (service1, config1) (service2, config2)...)
But I prefer what @danwinship suggested, simpler and don't have to deal with bash arrays.